examples/trials/nas_cifar10/config_ppo.yml

```diff
@@ -15,6 +15,8 @@ tuner:
     trials_per_update: 60
     epochs_per_update: 12
     minibatch_size: 10
+  #could use the No. 0 gpu for this tuner
```

Comment:
could you at least offer an example for multiply  GPU?

Reply:
sure, I will give an example. thx.